src/GNDStk/Node/test/many.test.cpp

```diff
@@ -18,14 +17,15 @@ SCENARIO("Testing GNDStk Node all()") {
          const auto &CEs = tree(reactionSuite,PoPs,chemicalElements);
          {
             // without "found"
-            auto ce = CEs.all("chemicalElement");
+            CHECK(CEs.has_many("chemicalElement"));
+            auto ce = CEs.many("chemicalElement");
             THEN("We find eight of them") {
                CHECK(ce.size() == 8);
             }
          } {
             // with "found"
             bool found = false;
-            auto ce = CEs.all("chemicalElement",found);
+            auto ce = CEs.many("chemicalElement",found);
```

Comment:
I know one(...) will return a reference to the node that was found. Does many(...) return a container of references, or a container of copies of the nodes that were found?

Reply:
It returns a container of copies. I'm not sure whether or not we'd actually want a container of references. If we did, I'd of course need to use something like `std::reference_wrapper`.

Reply:
Thinking about it, containers of `std::reference_wrapper`s may well make sense, for when users want to modify GNDS trees rather than just query them. This is probably a discussion we should have sometime soon.

Reply:
Yes, using reference wrappers would be nice but it is not urgent for now.

Reply:
Actually, if and when we're sure that containers of reference wrappers wouldn't have funny (in a bad way) semantics, this approach might resolve some questions I've been running into in designing a new node operator[] that's equipped for modifying existing nodes. So I suspect this issue will be revisited sooner rather than later, but probably not immediately.